src/components/WorkshopPage/WorkshopPage.js

```diff
@@ -277,6 +277,12 @@ function WorkshopPage() {
 					<Typography variant='h4' component='h1' className={classes.title}>
 						Workshops
 					</Typography>
+					<Typography variant='h5' component='h2'
+						style={{
+							paddingBottom: theme.spacing(isSmall ? 4 : 8)
+						}}>
+						Workshops coming soon, here are the ones from last year!
```

Comment:
We have a `<ComingSoon>` component that you can use instead of the "Workshops coming soon" text. You can see an example of how to use it in our [Schedule Page](https://github.com/uclaacm/hoth.uclaacm.com/blob/4ab158933a4947f3988d9534619bb282c2dcbc57/src/components/SchedulePage/Schedule.js#L42)